packages/frontend/components/Header.tsx

```diff
@@ -27,7 +53,83 @@ export default function Header() {
           smallScreen: "icon",
           largeScreen: "icon",
         }}
-      />
+      /> */}
+
+      {/* custom connect button */}
+      <ConnectButton.Custom>
+        {({ account, chain, openAccountModal, openChainModal, openConnectModal, authenticationStatus, mounted }) => {
+          // Note: If your app doesn't use authentication, you
+          // can remove all 'authenticationStatus' checks
+          const ready = mounted && authenticationStatus !== "loading";
+          const connected =
+            ready && account && chain && (!authenticationStatus || authenticationStatus === "authenticated");
+
+          return (
+            <div
+              {...(!ready && {
+                "aria-hidden": true,
+                style: {
+                  opacity: 0,
+                  pointerEvents: "none",
+                  userSelect: "none",
+                },
+              })}
+            >
+              {(() => {
+                if (!connected) {
+                  return (
+                    <button onClick={openConnectModal} type="button">
+                      Connect Wallet
+                    </button>
+                  );
+                }
+
+                if (chain.unsupported) {
+                  return (
+                    <button onClick={openChainModal} type="button">
+                      Wrong network
+                    </button>
+                  );
+                }
+
+                return (
+                  <div style={{ display: "flex", gap: 12 }}>
+                    <button onClick={openChainModal} style={{ display: "flex", alignItems: "center" }} type="button">
+                      {chain.hasIcon && (
+                        <div
+                          style={{
+                            background: chain.iconBackground,
+                            width: 12,
+                            height: 12,
+                            borderRadius: 999,
+                            overflow: "hidden",
+                            marginRight: 4,
+                          }}
+                        >
+                          {chain.iconUrl && (
+                            <img
```

Comment:
Check the linting. 
![image](https://user-images.githubusercontent.com/2486142/201665214-a041dcb7-0bcb-47ee-930b-6c915db6f06d.png)